WikiWatershed/model-my-watershed
# BiG-CZ: Branding Changes

## Overview

Update text in the splash page and the analyze page. Add more sponsor logos in the splash page. Move sponsor logos below main call to action in splash page. Update favicon.

Also tagging @ajrobbins for visual review.

Connects #2394 

### Demo

#### BiG-CZ

##### Splash Page

![image](https://user-images.githubusercontent.com/1430060/32390004-182d1c38-c0a3-11e7-85ad-bd8128de5275.png)

##### Draw Page

![image](https://user-images.githubusercontent.com/1430060/32461485-f764911c-c303-11e7-9f6a-82c369142f5b.png)

##### Analyze Page

![image](https://user-images.githubusercontent.com/1430060/32386577-17fd850a-c098-11e7-933e-bb2d276b3d7f.png)

##### Data Page

![image](https://user-images.githubusercontent.com/1430060/32388916-9624c676-c09f-11e7-9791-86ef76b88331.png)

#### MMW

##### Splash Page

![image](https://user-images.githubusercontent.com/1430060/32391496-b25a0cee-c0a8-11e7-9adc-716c885de135.png)

##### Draw Page

![image](https://user-images.githubusercontent.com/1430060/32386602-2a011528-c098-11e7-8971-cedd1c8738d9.png)

##### Analyze Page

![image](https://user-images.githubusercontent.com/1430060/32386616-345b6fe6-c098-11e7-9530-ac78f0b6f8c3.png)

## Testing Instructions

 * Check out this branch and run `manage.sh collectstatic` and `bundle.sh --debug --vendor`
 * Go to [:8000/?bigcz](http://localhost:8000/?bigcz) and ensure you see BiG-CZ specific text and branding
 * Go to [:8000/](http://localhost:8000/) and ensure you see WikiWatershed specific text and branding
 * Use [this document](https://docs.google.com/document/d/1oaS6YidSgJPsREBQcGgh70Q__aj56lBMQE4i5GbbdBU/edit#) rather than the issue description as reference for checking
